frontend/src/stores/ParticipantStore.js

```diff
@@ -161,10 +165,17 @@ export class ParticipantStore {
   @action setSidebarView = sidebarView => {
     this.sidebarView = sidebarView
   }
-  @action setSnackbarState = (message, options = { open: true }) => {
-    // we setup otions like this to maybe add more later
-    this.snackbarState.message = message
-    this.snackbarState.open = options.open
+  @action setSnackbarState = ({
```

Comment:
I like this approach better; (actually it was implemented like this from where i took it but in typescript). I designed it the other way  (with message, then options)  so it was a little easier to use and harder to send an empty message.  Since we are not using typescript, was there a reason you had to switch it?

I'm ok with leaving it because ultimately we wont be doing to much more. It would be cool to have generic "success", "something went wrong", etc messages as defaults instead of an empty string, but then we'd have to conditionally pick them based on the enum.

yea but curious why you changed it

Reply:
The deconstruction of the options object properties with default values doesn't work when the options object is a second parameter. All of the parameters have to get passed as a single object for this to work.

Reply:
```
  @action setSnackbarState = (message, options = {}) => {
    const severity = options.severity ? options.severity : SNACKBAR_SEVERITY.ERROR
    const open = options.open ? options.open : true
    // prevent visual change to severity or message when closing the snackbar
    if (open) {
      this.snackbarState.severity = severity
      this.snackbarState.message = message
    }
    this.snackbarState.open = open
  }
```
heres what i was talking about in the meeting
id rather have it like this  just to keep it easier to use, since there might have already been an mistake/error somewhere else lol